frontend/src/pages/org/crawl-detail.ts

```diff
@@ -451,14 +451,26 @@ export class CrawlDetail extends LiteElement {
   }
 
   private renderPanel(title: any, content: any) {
+    let panelContainer;
+    switch (this.sectionName) {
+      case "replay":
+        panelContainer = html`
+          <div class="flex-1 rounded-lg border overflow-hidden">${content}</div>
+        `;
+      break;
+      default:
+        panelContainer = html`
+          <div class="flex-1 rounded-lg border p-5">${content}</div>
```

Comment:
It would be preferable to handle this logic in the `class=` attribute on line 473, either using [`classMap`](https://lit.dev/docs/api/directives/#classMap) or
```
class=${`flex-1 rounded-lg border ${this.sectionName === "replay" ? "overflow-hidden" : "p-5"}`}
```